src/py/flwr/driver/driver_client_proxy.py

```diff
@@ -0,0 +1,161 @@
+        return cast(
+            common.GetPropertiesRes,
+            self._send_receive_msg(server_message_proto, timeout).get_properties_res,
+        )
+
+    def get_parameters(
+        self, ins: common.GetParametersIns, timeout: Optional[float]
+    ) -> common.GetParametersRes:
+        """Return the current local model parameters."""
+        server_message_proto: transport_pb2.ServerMessage = (
+            serde.server_message_to_proto(
+                server_message=common.ServerMessage(get_parameters_ins=ins)
+            )
+        )
+        return cast(
+            common.GetParametersRes,
+            self._send_receive_msg(server_message_proto, timeout).get_parameters_res,
+        )
+
+    def fit(self, ins: common.FitIns, timeout: Optional[float]) -> common.FitRes:
+        """Train model parameters on the locally held dataset."""
+        server_message_proto: transport_pb2.ServerMessage = (
+            serde.server_message_to_proto(
+                server_message=common.ServerMessage(fit_ins=ins)
+            )
+        )
+        return cast(
+            common.FitRes, self._send_receive_msg(server_message_proto, timeout).fit_res
+        )
+
+    def evaluate(
+        self, ins: common.EvaluateIns, timeout: Optional[float]
+    ) -> common.EvaluateRes:
+        """Evaluate model parameters on the locally held dataset."""
+        server_message_proto: transport_pb2.ServerMessage = (
+            serde.server_message_to_proto(
+                server_message=common.ServerMessage(evaluate_ins=ins)
+            )
+        )
+        return cast(
+            common.EvaluateRes,
+            self._send_receive_msg(server_message_proto, timeout).evaluate_res,
+        )
+
+    def reconnect(
+        self, ins: common.ReconnectIns, timeout: Optional[float]
+    ) -> common.DisconnectRes:
+        """Disconnect and (optionally) reconnect later."""
+        return common.DisconnectRes(reason="")  # Nothing to do here (yet)
+
+    def _send_receive_msg(
+        self, server_message: transport_pb2.ServerMessage, timeout: Optional[float]
+    ) -> transport_pb2.ClientMessage:
+        task_ins = task_pb2.TaskIns(
+            task_id="",
+            group_id="",
+            workload_id="",
+            task=task_pb2.Task(
+                producer=node_pb2.Node(
+                    node_id=0,
+                    anonymous=True,
+                ),
+                consumer=node_pb2.Node(
+                    node_id=self.node_id,
+                    anonymous=self.anonymous,
+                ),
+                legacy_server_message=server_message,
+            ),
+        )
+        push_task_ins_req = driver_pb2.PushTaskInsRequest(task_ins_list=[task_ins])
+        push_task_ins_res = self.driver.push_task_ins(req=push_task_ins_req)
+        time.sleep(SLEEP_TIME)
+        task_ids = [task_id for task_id in push_task_ins_res.task_ids if task_id != ""]
+        all_task_res = []
+
+        if len(task_ids) == 0:
```

Comment:
Also check for >1, which should never happen (since we always only schedule one `TaskIns`)